Makefile

```diff
@@ -17,15 +17,15 @@
 # Build tunables
 
 # When updating these defaults, be sure to check that ALL_BUILDABLE_FLAVORS is updated
-# CEPH_VERSION,ARCH,OS_NAME,OS_VERSION,BASEOS_REG,BASEOS_REPO,BASEOS_TAG
+# CEPH_VERSION,BASEOS_REPO,BASEOS_TAG
 FLAVORS ?= \
-	luminous,x86_64,ubuntu,16.04,_,ubuntu,16.04 \
-	jewel,x86_64,ubuntu,16.04,_,ubuntu,16.04 \
-	jewel,x86_64,ubuntu,14.04,_,ubuntu,14.04 \
-	kraken,x86_64,ubuntu,16.04,_,ubuntu,16.04 \
-	luminous,x86_64,centos,7,_,centos,7 \
-	jewel,x86_64,centos,7,_,centos,7 \
-	kraken,x86_64,centos,7,_,centos,7 \
+	luminous,ubuntu,16.04 \
```

Comment:
Why does ARCH has vanished ? We need it for arm64 support.

Reply:
ARCH hasn't vanished. It is computed automatically from the host arch.

Reply:
OK

Reply:
I'll have to add support for crossbuilds later. I'm not sure crossbuilds would actually have worked in the previous schema anyway.

Reply:
Tried it but failed. Looks like we'll need dedicated build node and so the implicit ARCH would be ok.

Reply:
I talked with Bassam a bit yesterday about this. I think it's possible to do crossbuilds via make, but it'll require work to support and make sure it works.